std/container/rbtree.d

```diff
@@ -1778,6 +1818,11 @@ pure unittest
 /++
     Convenience function for creating a $(D RedBlackTree!E) from a list of
     values.
+
+    Params:
+        allowDuplicates =  Whether duplicates should be allowed (optional, default: false)
+        less = predicate to sort by (optional)
+        elems = elements to insert into the rbtree (can be a range or variadic arguments)
```

Comment:
Well, the parameter is called `range` though.

Maybe name the documented parameter `elems/range`? Or rename actual function parameter `elems`?